common/djangoapps/edraak_dummy/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import url
+from edraak_dummy.views import SetCSRFDummyView
+
+urlpatterns = [
+    url(
+        '^edraak_set/csrf',
+        SetCSRFDummyView.as_view(),
```

Comment:
Thanks @shadinaif, Could you please remove the `edraak_dummy` app? It was there to solve a bug in Dogwood that Ali and I managed to resovle in Hawthorn.

Sorry for making you to go into the trouble of migrating it.

Reply:
Done